library/std/src/sys/unix/thread.rs

```diff
@@ -271,6 +293,27 @@ impl Drop for Thread {
     }
 }
 
+#[derive(Debug)]
+pub struct BuilderOptions {
+    /// The spawned thread's priority value
+    #[cfg(target_os = "horizon")]
+    pub(crate) priority: Option<i32>,
```

Comment:
on the OS side these `None`s are represented as sentinel values, aren’t they? What is the value in being able to differentiate between `None` and the sentinel value?

Reply:
If it's None we use the current thread's priority.

Reply:
The OS side is also in Rust (pthread-3ds), so the sentinal value is also None.